app/models/annotation/nml/NmlParser.scala

```diff
@@ -56,14 +56,15 @@ object NmlParser extends LazyLogging with ProtoGeometryImplicits {
         val editPosition = parseEditPosition(parameters \ "editPosition").getOrElse(SkeletonTracingDefaults.editPosition)
         val editRotation = parseEditRotation(parameters \ "editRotation").getOrElse(SkeletonTracingDefaults.editRotation)
         val zoomLevel = parseZoomLevel(parameters \ "zoomLevel").getOrElse(SkeletonTracingDefaults.zoomLevel)
-        val userBoundingBox = parseUserBoundingBox(parameters \ "userBoundingBox")
+        val userBoundingBox = parseBoundingBox(parameters \ "userBoundingBox")
+        val boundingBox = parseBoundingBox(parameters \ "boundingBox")
 
         logger.debug(s"Parsed NML file. Trees: ${trees.size}, Volumes: ${volumes.size}")
 
         if (volumes.size >= 1) {
           (Right(VolumeTracing(None, BoundingBox.empty, time, dataSetName, editPosition, editRotation, ElementClass.uint32, None, 0, 0, zoomLevel), volumes.head.location), description)
         } else {
-          (Left(SkeletonTracing(dataSetName, trees, time, None, activeNodeId,
+          (Left(SkeletonTracing(dataSetName, trees, time, boundingBox, activeNodeId,
```

Comment:
Why should the boundingBox be part of the NML? I thought this belongs rather to the dataset than to a tracing?

Reply:
In https://github.com/scalableminds/webknossos/blob/master/webknossos-datastore/proto/SkeletonTracing.proto in line 63 you can see that the bounding box is already part of a skeletonTracing. 
The reason why we want to write the bounding box to the nml is that after you've downloaded the nml of a SkeletonTracing and then you are uploading it again, you dont't want to lose the bounding box. (see: https://discuss.webknossos.org/t/bounding-box-should-be-persisted-to-parsed-from-nml/966)

Reply:
Ah, thanks! :+1:  I wasn't aware of that layer.